src/calculateRentalCost.js

```diff
@@ -5,6 +5,18 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  let totalCost = cost * days;
+
+  if (days >= 7) {
+    return (totalCost -= 50);
+  }
+
+  if (days >= 3) {
+    return (totalCost -= 20);
+  }
```

Comment:
Same as above, avoid using 'else' after an 'if' with a 'return' as it is unnecessary and can be omitted to simplify the code.